### DISCUSSION TITLE
HW 1 Question

### DISCUSSION THREAD

**Discussion by shawnveilleux** [cooperrc/advanced-dynamics#23] [General]
I understand that the equation of motion for the pendulum with a_x = 0 m/s^2 is 
![image](https://github.com/cooperrc/advanced-dynamics/assets/143618436/75869d58-5a9c-43a3-9fcc-c78fb0d6acbc)
and with an a_x of 5 m/s^2 it is.
![image](https://github.com/cooperrc/advanced-dynamics/assets/143618436/f7241011-798d-40d3-947e-63afa4cac390)
I am able to solve for θ(t) when a_x = 0 using principles of differential equations however when attempting to solve the one with the a_x term, I can't seem to yield any results. I've tried online solvers as well as Matlab and everything says there are no solutions.
My question is, am I going in the right direction? Or are the answers not derived from those equations of motion?

**Comment by cooperrc** [comment]
Nice research here, 

even in the first case, 

$\ddot{\theta} = -\frac{g}{r}\sin\theta$

there is no exact solution, but if we make a small-angle approximation, 

$\theta<<1\rightarrow \sin\theta\approx\theta$ and $\cos\theta\approx 1$

then, you can use the linear ODE solution, 

$\theta(t) = A\cos\omega t + B\sin\omega t$

for the second part where $a_x =5$

you have $\times 2$ parts to your solution, 

$\theta(t) = \theta_H(t) + \theta_P(t)$

where the H and P are homogeneous and particular solutions to the ODE

**Reply by shawnveilleux** [reply]
That helped me significantly and I was able to get the answers. Thank you!
